libgit2-sys/lib.rs

```diff
@@ -600,6 +603,7 @@ pub enum git_ref_t {
 }
 pub use git_ref_t::*;
 
+enum_from_primitive!{
```

Comment:
Ah, excellent!  If we already have a type for that then there's no problem making the change.